client/state/stats/lists/utils.js

```diff
@@ -14,3 +16,24 @@ import toPairs from 'lodash/toPairs';
 export function getSerializedStatsQuery( query = {} ) {
 	return JSON.stringify( sortBy( toPairs( query ), ( pair ) => pair[ 0 ] ) );
 }
+
+export const Parsers = {
```

Comment:
Perhaps a docs update is in order.  Thinking about it a bit more, I do remember seeing this pattern and I'm not sure I like either of my suggestions better.